src/test/test_candlesticks.py

```diff
@@ -66,28 +66,94 @@ def test_shorten_fails_if_non_int_passed(self):
 
         self.assertRaises(TypeError, candles.shorten, required_length)
 
-    def test_diff_up(self):
+    def test_position_1m_(self):
         df = pd.DataFrame([[0], [1]], columns=['Signal'])
```

Comment:
Change this to position - technically my fault

Reply:
should be good